pkgs/misc/vim-plugins/generated.nix

```diff
@@ -6699,18 +6699,6 @@ let
     meta.homepage = "https://github.com/tpope/vim-sleuth/";
   };
 
-  vim-slime = buildVimPluginFrom2Nix {
```

Comment:
Why got this removed?

Reply:
For some reason running `update.py` returns `HTTP Error 404` for `vim-slime` alone. The author must have removed the plugin from `vim-plugin-names` and later added it back and amended the commit (but the plugin got removed from `generated.nix` and it may have been a bit unintuitive behavior).